stardog/templates/post-install-job.yaml

```diff
@@ -35,41 +35,58 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         env:
         - name: STARDOG_JAVA_ARGS
-          value: -XX:ParallelGCThreads=1
+          value: -Dstardog.default.cli.server=http://{{ include "stardog.fullname" . }}:5820 -XX:ParallelGCThreads=1
         - name: STARDOG_ADMIN_PW
           valueFrom:
             secretKeyRef:
               name: {{ include "stardog.fullname" . }}-user-passwords
               key: admin
         - name: STARDOG_SERVER_URL
           value: http://{{ include "stardog.fullname" $ }}:5820
-        {{- range .Values.stardog.users }}
-        - name: {{ .name | upper | replace "-" "_" }}
-          valueFrom:
-            secretKeyRef:
-              name: {{ include "stardog.fullname" $ }}-user-passwords
-              key: {{ .name }}
-        {{ end }}
         command:
-        - /bin/sh
+        - /bin/bash
         - -ce
         - |
-          echo "Testing stardog connectivity to ${STARDOG_SERVER_URL}"
-          until $(curl -o /dev/null -s -f "${STARDOG_SERVER_URL}/admin/alive"); do sleep 1 && echo "."; done
+          script="/var/opt/stardog/stardog-wrapper.sh"
+          echo "Testing stardog connectivity to ${STARDOG_SERVER_URL}..."
+          until $(curl -o /dev/null -s -f "${STARDOG_SERVER_URL}/admin/alive"); do sleep 1; done
+          echo "Stardog is available now"
 
-          {{- if .Release.IsInstall }}
-          echo Disable anonymous user
-          /opt/stardog/bin/stardog-admin --server "${STARDOG_SERVER_URL}" user disable anonymous
-          echo Changing admin password
-          /opt/stardog/bin/stardog-admin --server "${STARDOG_SERVER_URL}" user passwd --new-password "$STARDOG_ADMIN_PW"
-          echo
-          {{- end }}
```

Comment:
@srueg In the previous version both were in IsInstall block, should that be changed?

Reply:
If the call doesn't fail if disabling an already disabled user, yes, I'd propose to run it always.

Reply:
Need to check that first

Reply:
So, it works repeatedly with exit code 0. But then my question is, why not run the command to set the admin password every time as well? Basically remove the `if IsInstall` condition?